go.mod

```diff
@@ -61,6 +61,7 @@ require (
 	github.com/mitchellh/go-wordwrap v1.0.1 // indirect
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/mypurecloud/platform-client-sdk-go/v94 v94.0.1 // indirect
```

Comment:
Need to run `go mod tidy` to get rid of the dependency. GitHub Actions can't seem to handle having two versions of our platform sdk at the same time